### PR TITLE
remove-browser-console-clear

### PR DESCRIPTION
**Pull request by AndreTheHunter**
- [x] https://github.com/jesims/bindle/pull/38

### CODE REVIEW

File: src/io/jesi/customs/runner/browser.cljs [hunk 1]
```diff
@@ -36,7 +36,6 @@
       (test/inc-report-counter! :fail))))
 
 (defn start []
-  (js/console.clear)
```
Comment (AndreTheHunter) [inline src/io/jesi/customs/runner/browser.cljs:39]: Removed
Users can still do
```clj
(defn ^:dev/after-load clear-console []
  (js/console.clear))
```
if they want to clear the console

File: customs.iml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module cursive.leiningen.project.LeiningenProjectsManager.displayName="io.jesi/customs:1.1.2" cursive.leiningen.project.LeiningenProjectsManager.isLeinModule="true" type="JAVA_MODULE" version="4">
+<module cursive.leiningen.project.LeiningenProjectsManager.displayName="io.jesi/customs:1.2.0" cursive.leiningen.project.LeiningenProjectsManager.isLeinModule="true" type="JAVA_MODULE" version="4">
```
Comment (axrs) [inline customs.iml]: IMls refresh if version changed?

**Review by axrs — CHANGES_REQUESTED** [review]
And the checkbox items

File: src/io/jesi/customs/spy.cljc [hunk 3]
```diff
@@ -36,30 +37,49 @@
        (println ~@(let [line (line-number file form)]
                     (trans-reduce
                       (fn [col form]
-                        (doto col
-                          (conj! (str (-name form) \:))
-                          (conj! `(pr-str ~form))))
+                        (-> col
+                            (conj! (str (-name form) ":"))
+                            (conj! `(pr-str ~form))))
```
Comment (AndreTheHunter) [inline src/io/jesi/customs/spy.cljc:42]: Fix potential bug

File: test/io/jesi/customs/spy_test.cljc [hunk 4]
```diff
@@ -128,6 +129,15 @@
     #?(:clj (testing "is a macro"
               (bp/macro? `spy/peek)))
 
+    #?(:clj (comment                                        ;Fails since Actual is a vector (not a list)
```
Comment (AndreTheHunter) [inline test/io/jesi/customs/spy_test.cljc:132]: Not sure why it's in a vector...

File: customs.iml (hunk 2 above)
Comment (AndreTheHunter) [reply customs.iml]: Updated `.iml`

File: src/io/jesi/customs/runner/browser.cljs (hunk 1 above)
Comment (AndreTheHunter) [reply src/io/jesi/customs/runner/browser.cljs:39]: Added to Changelog

**Review by axrs — DISMISSED** [review]

**Review by axrs — APPROVED** [review]

**Pull request merged**